node-notes-cli/app.js

```diff
@@ -0,0 +1,45 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+async function parse() {
```

Comment:
Nice that you encapsulated this into a function. I like it.

I think `parse` is an awkward name, though. The function name tells _how_ it is doing the work, but it doesn't tell _what_ it is doing. Something like `readData` or `loadData` would be more explanatory.